ddpui/utils/warehouse/client/schema_queries.py

```
@@ -0,0 +1,30 @@
+from ddpui.utils.warehouse.client.warehouse_interface import WarehouseType
+
+
+SCHEMA_QUERIES: dict[str, str] = {
+    WarehouseType.POSTGRES.value: """
+        SELECT schema_name
+        FROM information_schema.schemata
+        WHERE schema_name NOT IN ('pg_catalog', 'information_schema', 'airbyte_internal')
+          AND schema_name NOT LIKE 'pg\_%'
```

Ruff (0.15.10)

[warning] Invalid escape sequence: `\_`

Use a raw string literal

(W605)